OsmAnd/src/net/osmand/plus/plugins/osmedit/OsmEditingPlugin.java

```diff
@@ -343,8 +343,12 @@ public void registerMapContextMenuActions(@NonNull MapActivity mapActivity,
 			} else {
 				amenity = ((TransportStop) selectedObj).getAmenity();
 			}
-			PoiType poiType = amenity.getType().getPoiTypeByKeyName(amenity.getSubType());
-			isEditable = !amenity.getType().isWiki() && poiType != null && !poiType.isNotEditableOsm();
+			if (settings.isInternetConnectionAvailable() && ObfConstants.isOsmUrlAvailable(amenity)) {
+				isEditable = true;
+			} else {
+				PoiType poiType = amenity.getType().getPoiTypeByKeyName(amenity.getSubType());
+				isEditable = !amenity.getType().isWiki() && poiType != null && !poiType.isNotEditableOsm();
```

Comment:
Do we need to ignore amenity.getType().isWiki(), or only the second part with poiType != null && !poiType.isNotEditableOsm()?